addons/website/views/website_templates.xml

```diff
@@ -364,7 +364,7 @@
                     </button>
                 </li>
             </ul>
-            <div t-attf-class="offcanvas #{_side if _side else 'offcanvas-end'} o_navbar_mobile" id="top_menu_collapse_mobile">
+            <div t-attf-class="offcanvas #{_side if _side else 'offcanvas-end'} o_navbar_mobile navbar-light" id="top_menu_collapse_mobile">
```

Comment:
I'm not convinced of this solution for 2 reasons:
- Users won't get the fix without updating their website (as usual, we need to avoid XML fixes in stable).
- The "navbar-light" is already present on the `o_header_mobile` so it's likely useless to add it again on one of its children.

What I would do is rather add a selector `.navbar-light .o_navbar_mobile` for the `.navbar-light` rule :
```scss
.navbar-light, .navbar-light .o_navbar_mobile {
```
or another idea, applying the mixin to the `o_navbar_mobile` element (maybe it's safer because it will only change the background-image compared to how it is now)
```scss
.navbar-light .o_navbar_mobile {
    @include o-add-gradient('menu-gradient');
}
```
I'll let you decide what is best 😉